api/users/urls.py

```diff
@@ -13,5 +13,6 @@
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
     url(r'^(?P<user_id>\w+)/preprints/$', views.UserPreprints.as_view(), name=views.UserPreprints.view_name),
     url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name),
+    url(r'^(?P<user_id>\w+)/files/$', views.UserQuickFiles.as_view(), name=views.UserQuickFiles.view_name),
```

Comment:
This format was discussed with @brianjgeiger over the best way to deal with the upload link existing on a user page over a file detail page for ease of access. This is the way he suggested I format it, happy to change it if opinions are now different